src/ui/views/session/discover/DiscoverView.tsx

```diff
@@ -0,0 +1,143 @@
+import { useState } from "react";
+import { Room } from "@thirdroom/hydrogen-view-sdk";
+
+import { Content } from "../../../atoms/content/Content";
+import { Header } from "../../../atoms/header/Header";
+import { HeaderTitle } from "../../../atoms/header/HeaderTitle";
+import ExploreIC from "../../../../../res/ic/explore.svg";
+import ChevronRightIC from "../../../../../res/ic/chevron-right.svg";
+import { Icon } from "../../../atoms/icon/Icon";
+import { Window } from "../../components/window/Window";
+import { DiscoverHome } from "./DiscoverHome";
+import { SegmentControl } from "../../../atoms/segment-control/SegmentControl";
+import { SegmentControlItem } from "../../../atoms/segment-control/SegmentControlItem";
+import { usePowerLevels } from "../../../hooks/usePowerLevels";
+import { useHydrogen } from "../../../hooks/useHydrogen";
+import { DiscoverAdmin } from "./DiscoverAdmin";
+import { DiscoverAll } from "./DiscoverAll";
+import { DiscoverCreator } from "./DiscoverCreator";
+import "./DiscoverView.css";
+
+export enum RepositoryEvents {
+  FeaturedWorlds = "tr.repository_room.featured_worlds",
+  FeaturedRooms = "tr.repository_room.featured_rooms",
+  FeaturedScenes = "tr.repository_room.featured_scenes",
+  Scene = "tr.repository_room.scene",
+}
+
+enum DiscoverTab {
+  Home = "Home",
+  Admin = "Admin",
+  Creator = "Creator",
+}
+
+export function DiscoverView({ room }: { room: Room }) {
+  const { session } = useHydrogen(true);
+  const { getPowerLevel, canSendStateEvent } = usePowerLevels(room);
+  const [discoverTab, setDiscoverTab] = useState<DiscoverTab>(DiscoverTab.Home);
+  const [loadEvents, setLoadEvents] = useState<RepositoryEvents>();
```

Comment:
This should be `loadEvent` instead of `loadEvents` as it is singular not plural.